src/main/webapp/script.js

```diff
@@ -72,7 +73,18 @@ function createResultCard(name, address, photos, rating, totalCardCount) {
 
   const imageDiv = document.createElement('div');
   imageDiv.className = 'results-image';
-  // TODO: Link image
+
+  const imageElement = document.createElement('img');
+  
+  if ((photos != null) && (photos.length > 0)) {
+    let photoReference = photos[0].photoReference;
+    const KEY = 'AIzaSyDDIsG-SJAZ69ZoOecmfbXOB7ZIS4pZkAw';
```

Comment:
Should this be 'Redacted'? I don't think we want to expose these keys to the client.

Reply:
Oh shoot I didn't even realize that was in there

Reply:
Removed